app/Controllers/feedController.php

```diff
@@ -198,6 +198,7 @@ public function addAction() {
 				'curl_params' => empty($opts) ? null : $opts,
 			);
 			$attributes['ssl_verify'] = Minz_Request::paramTernary('ssl_verify');
+			$attributes['ipresolve'] = Minz_Request::paramTernary('ipresolve');
```

Comment:
Instead of a boolean (true / false), I would prefer a string such as `IPv4` or `IPv6`.